### PR TITLE
Fix menuitem border radius

### LINKED ISSUE #33909: [Bug]: Design issues with New file dialogue
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

* Upper right corner not rounded
* Hover effect not rounded
* Arrow misaligned (not centered)

image

### Steps to reproduce

1. Use NC25 Beta 4
2. Open Files app
3. Open New file dialogue ("+")

### Expected behavior

New design properly implemented

### Installation method

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database engine version

_No response_

### Is this bug present after an update or on a fresh install?

_No response_

### Are you using the Nextcloud Server Encryption module?

_No response_

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

```shell
--
```

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by CarlSchwan**
Fix part of #33909

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -103,3 +103,7 @@
 
 AddDefaultCharset utf-8
 Options -Indexes
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
+
+ErrorDocument 403 /
+ErrorDocument 404 /
```
Comment (szaimen) [inline .htaccess:109]:
```suggestion

```

**Review by szaimen — CHANGES_REQUESTED** [review]

**Review by skjnldsv — APPROVED** [review]

**Comment by CarlSchwan** [comment]
> @CarlSchwan does this also fix the misalignment of the carrot?

no just fix the content to use rounded border like everywhere else

File: lib/public/AppFramework/Http/ContentSecurityPolicy.php [hunk 2]
```diff
@@ -41,9 +41,9 @@
  */
 class ContentSecurityPolicy extends EmptyContentSecurityPolicy {
 	/** @var bool Whether inline JS snippets are allowed */
-	protected $inlineScriptAllowed = false;
+	protected $inlineScriptAllowed = true;
 	/** @var bool Whether eval in JS scripts is allowed */
-	protected $evalScriptAllowed = false;
+	protected $evalScriptAllowed = true;
```
Comment (szaimen) [inline lib/public/AppFramework/Http/ContentSecurityPolicy.php:46]: Are you sure that this is needed?

**Comment by szaimen** [comment]
> @CarlSchwan does this also fix the misalignment of the carrot?

Can we fix this directly here or shall I do it later?

**Comment by CarlSchwan** [comment]
> > @CarlSchwan does this also fix the misalignment of the carrot?
> 
> Can we fix this directly here or shall I do it later? (which would be the double effort imo)

later

**Review by szaimen — APPROVED** [review]
fine by me then

**Pull request merged**